Packs/DB2/Integrations/DB2/DB2.yml

```diff
@@ -0,0 +1,71 @@
+category: Database
+commonfields:
+  id: DB2
+  version: -1
+configuration:
+- display: Database host
+  name: host
+  required: true
+  type: 0
+- display: Port
+  name: port
+  required: false
+  type: 0
+- display: Database Name
+  name: dbname
+  required: false
+  type: 0
+- display: Username
+  name: credentials
+  required: true
+  type: 9
+- display: 'Connection Arguments (ex: arg1=val1&arg2=val2)'
+  name: connect_parameters
+  required: false
+  type: 0
+- display: Use an SSL connection
+  name: ssl_connect
+  required: false
+  type: 8
+- display: Use Persistent Connection
+  name: use_persistent
+  required: false
+  type: 8
+contentitemexportablefields:
+  contentitemfields:
+    definitionid: ""
+    fromServerVersion: ""
+    itemVersion: ""
+    packID: 94806677-b9ef-4012-8caa-e000b7ad1979
+    propagationLabels:
+    - all
+    toServerVersion: ""
+description: Integration to provide connectivity to IBM DB2 using the python ibm_db2
+  library.
+display: DB2
+name: DB2
+script:
+  commands:
+  - arguments:
+    - description: The DB2 query to run
+      name: query
+      required: true
+    - defaultValue: "50"
+      description: The maximum number of results
+      name: limit
+    - defaultValue: "0"
+      description: The offset at which to start the result
+      name: skip
+    - description: A comma separated list of names which will be replaced in query
+        having ':<name>'
+      name: bind_variables_name
+    - description: A comma separated list of values corresponding to `bind_variables_name`
+        or values in order of '?' mark to be replaced in query
+      name: bind_variables_values
+    description: Running a DB2 query command
+    name: db2-query
+  dockerimage: ibm_db2:latest
```

Comment:
yes, as the `ibm_db` module is used in this integration.

Reply:
alright, so we must wait for the docker image PR to be merged before we can merge this PR. Mainly the build issues are because of that. it can't find the docker image.